src/jmcomic/__init__.py

```
@@ -4,6 +4,7 @@
 
 __version__ = '2.7.2'
 
+from .jm_task_context import *
```

Ruff (0.16.0)

[error] `from .jm_task_context import *` used; unable to detect undefined names

(F403)